index.js

```diff
@@ -1,18 +1,18 @@
 module.exports = function (data, replacer, space, opts) {
-  var redactedKeys = opts && opts.redactedKeys ? opts.redactedKeys : []
-  var redactedPaths = opts && opts.redactedPaths ? opts.redactedPaths : []
+  const redactedKeys = opts && opts.redactedKeys ? opts.redactedKeys : []
```

Comment:
We can't make these changes because this package isn't compiled with babel, so using `const` & `let` is breaking for users on Node < 6

Reply:
The latest version of the`standard` code formatter that is used here doesn't support configuring the rules. I could attempt to switch the formatting to use ESLint with a config that doesn't change these var declarations?

However I'm unsure if this is worthwhile given that Node 6 was end of life over half a decade ago. What are your thoughts on this @imjoehaines?